casts/cards/src/UICollectionCard/types.ts

```diff
@@ -0,0 +1,27 @@
+import type { ViewStyle, ImageSourcePropType } from 'react-native';
+
+export type ContentType = 'Image' | 'Video' | 'Unknown';
+
+export type UICollectionCardProps = {
+    contentType: ContentType;
```

Comment:
For now it's ok as we use a single element of the source array in the Preview, but in subsequent iterations we should be also able to process an array of heterogeneous media files formats (unfortunately we can have an image, gif and video in one collection)